.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,24 @@
+# Run secret-dependent integration tests only after /ok-to-test approval
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - 'README.md'
+      - 'CHANGELOG.md'
+name: unit
```

Comment:
these are not unit tests

Reply:
fair point. renamed file to test-unit-and-integration.yml